ci/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"dagger.io/dagger"
+)
+
+const (
+	API_VERSION = "v1"
+
+	Go2WASM   = "go1.21"
```

Comment:
Go constants should always be

CamelCase

not ALLCAPS.  If you find code where I don't follow that rule, please fix it.

Reply:
thanks for the correction. I searched for naming convention in Stack Overflow and got different answers, which confused me. Now I realize I should have followed the official instructions.

Reply:
thanks for the correction. I searched for the naming convention in Stack Overflow and got different answers, which confused me. Now I realize I should have followed the official instructions.